dss/events/__init__.py

```diff
@@ -14,22 +15,15 @@
 from dss.storage.identifiers import TOMBSTONE_SUFFIX
 from dss.util.version import datetime_from_timestamp
 
-# TODO Support "reindex" for event journals:
-#      1) attempt read from "old" prefix
-#      2) attempt read from "new" prefix
-#      3) write to "new" prefix
-#      4) Prevent journaling on "new" prefix until all ald events are added
-
 # TODO: What happens when an event is recorder with timestamp erlier tha latest journal?
 
 @lru_cache(maxsize=2)
 def get_bundle_metadata_document(replica: Replica,
                                  key: str,
                                  flashflood_prefix: str=None) -> dict:
     fqid = key.split("/", 1)[1]
-    ff = FlashFlood(resources.s3,  # type: ignore
-                    Config.get_flashflood_bucket(),
-                    flashflood_prefix or replica.flashflood_prefix)
+    pfx = flashflood_prefix or replica.flashflood_prefix_read
```

Comment:
Why would a user wish to set this?  (I'm probably just arguing for a docstring explaining the argument here)

Reply:
Operators may wish to choose the prefix during administrative actions.